workos/user_management.py

```diff
@@ -540,3 +541,72 @@ def send_magic_auth_code(
         )
 
         return WorkOSUser.construct_from_response(response).to_dict()
+
+    def enroll_auth_factor(
+        self,
+        user,
+        type,
+        totp_issuer=None,
+        totp_user=None,
+    ):
+        """Enrolls a user in a new auth factor.
+
+        Kwargs:
+            user (str): The unique ID of the User to be enrolled in the auth factor.
+            type (str): The type of factor to enroll (Only option available is 'totp').
+            totp_issuer (str): Name of the Organization (Optional)
+            totp_user (str): Email of user (Optional)
+
+        Returns:
+            dict: AuthenticationChallengeAndFactor response from WorkOS.
+        """
+
+        if type not in ["totp"]:
+            raise ValueError("Type parameter must be 'totp'")
+
+        headers = {}
+
+        payload = {
+            "type": type,
+            "totp_issuer": totp_issuer,
+            "totp_user": totp_user,
+        }
+
+        response = self.request_helper.request(
+            USER_AUTH_FACTORS_PATH.format(user),
+            method=REQUEST_METHOD_POST,
+            headers=headers,
+            params=payload,
+            token=workos.api_key,
+        )
+
+        return WorkOSAuthenticationChallengeAndFactor.construct_from_response(
+            response
+        ).to_dict()
+
+    def list_auth_factors(
+        self,
+        user,
```

Comment:
do we tend to use params like `user` or `user_id` in the python SDK more? what about within the user management module?

Reply:
It seems to be more like `user`, `connection`, `organization`, etc. It definitely seems like doing the `_id` would be better, but that is the convention we're currently using elsewhere:

https://github.com/workos/workos-python/blob/fb0e087584c47f3fdc3ac29e137733b59fe2a15a/workos/user_management.py#L75

https://github.com/workos/workos-python/blob/fb0e087584c47f3fdc3ac29e137733b59fe2a15a/workos/sso.py#L175

https://github.com/workos/workos-python/blob/fb0e087584c47f3fdc3ac29e137733b59fe2a15a/workos/organizations.py#L157

Reply:
and we use organization_id here... but that's a list endpoint https://github.com/workos/workos-python/blob/fb0e087584c47f3fdc3ac29e137733b59fe2a15a/workos/user_management.py#L93

Reply:
blegh! lol

Reply:
yikes 🙈 I'm ok with starting to be more opinionated and go the `*_id` route

Reply:
yeah how about within the UM module we go w/ `*_id`?

fwiw we were also inconsistent at the API layer but for UM have made all params `*_id` (and that'll be the plan for new APIs going forward)

Reply:
cool cool - I'll make those changes throughout the module now 👍

Reply:
I made those changes throughout the UM module, just to remind myself for later changes